front/app/components/AssignBudgetControl/index.tsx

```diff
@@ -3,7 +3,7 @@ import {
   isNilOrError,
   capitalizeParticipationContextType,
 } from 'utils/helperUtils';
-import { openVerificationModal } from 'containers/App/VerificationModal/verificationModalEvents';
+import { openVerificationModal } from 'containers/Authentication/VerificationModal/verificationModalEvents';
```

Comment:
We shouldn't import directly from other containers. Can you move this event to the global events folder, same as `openSignUpInModal`.